definitions/mlops-machine_learning_model/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: MLOPS
+type: MACHINE_LEARNING_MODEL
+configuration:
+  entityExpirationTime: QUARTERLY
+  alertable: true
```

Comment:
Hi @neta-signifai do you want this entity to be alertable? if so we'd need to make a request to NerdGraph to make the domain MLOPS alertable. (cc: @srvaroa )

Reply:
we do